conans/client/conf/config_installer.py

```diff
@@ -273,10 +278,14 @@ def is_config_install_scheduled(api):
     config_install_file = cache.config_install_file
     if interval is not None:
         if os.path.exists(config_install_file):
-            timestamp = os.path.getmtime(config_install_file)
-            sched = datetime.fromtimestamp(timestamp, tz=gettz())
-            sched += interval
-            now = datetime.now(gettz())
-            return now > sched
+            if _load_configs(config_install_file):
+                timestamp = os.path.getmtime(config_install_file)
```

Comment:
This should happen always, what if there is one broken one not removed?

Reply:
The scheduler only checks if the task is scheduled and if a config is there, doesn't matter if it's broken or not. The method `configuration_install` (at same file) is the judge, it will consume the configs file.

I believe the scheduler should be stupid, only run a task if available and the interval passed. Otherwise, it will need to "steal" the `configuration_install` action too.